apps/src/appMain.js

```diff
@@ -125,6 +125,7 @@ export default function (app, levels, options) {
         next();
       }
     }
+    logToCloud.reportPageSize();
```

Comment:
Out of curiosity, what types of things do we log to New Relic vs. AWS Firehose?

Reply:
I don't know if we have a general rule. In this case, the rest of the performance data I'm using is already in NewRelic, so I decided to log page sizes there as well.